spec/collectors/statsd.rb

```diff
@@ -32,6 +32,16 @@ module Collectors
         it 'records a single result' do
           expect(collector.results.count).to eq 1
         end
+
+        it 'Converts test description to 8 character hash' do
+          str = collector.formatDesc('test')
+          expect(str.length).to eq(8)
```

Comment:
The hash is the same for unchanged string, so the value could be computed once and checked then in the test case

Reply:
Not fully clear what you mean by this.  Compute the string in a before block and run assertion only within test?